apps/asap-server/src/entities/user.ts

```diff
@@ -6,17 +6,32 @@ export interface CMSUser {
   data: {
     displayName: { iv: string };
     email: { iv: string };
-    firstName: { iv: string };
-    middleName: { iv: string };
-    lastName: { iv: string };
-    jobTitle: { iv: string };
-    orcid: { iv: string };
-    institution: { iv: string };
+    firstName?: { iv: string };
```

Comment:
Is there any particular reason to make this optional? We should consider redefining the type for the particular use-case with need this flexibility.

Reply:
I just updated it to reflect what is mandatory in the createSchema.